src/WebConnector/WebConnector.csproj

```diff
@@ -1,117 +1,21 @@
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Web.Services" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Asynchronous\IAuthenticator.cs" />
-    <Compile Include="Asynchronous\IStepQueryResponse.cs" />
-    <Compile Include="Asynchronous\IVersionValidator.cs" />
-    <Compile Include="Asynchronous\StepQueryResponseBase.cs" />
-    <Compile Include="Asynchronous\StepQueryWithIterator.cs" />
-    <Compile Include="AuthenticatedTicket.cs" />
-    <Compile Include="Synchronous\IVersionValidator.cs" />
-    <Compile Include="IStepQueryResponseBase.cs" />
-    <Compile Include="Synchronous\IAuthenticator.cs" />
-    <Compile Include="LogDirection.cs" />
-    <Compile Include="LogMessageType.cs" />
-    <Compile Include="Synchronous\StepQueryResponseBase.cs" />
-    <Compile Include="Synchronous\StepQueryWithIterator.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="QbManagerBase.cs" />
-    <Compile Include="QbSyncException.cs" />
-    <Compile Include="Asynchronous\QbWebConnectorSvc.cs">
-      <SubType>Component</SubType>
-    </Compile>
-    <Compile Include="Services\QbConnectorAsync.asmx.cs">
-      <DependentUpon>QbConnectorAsync.asmx</DependentUpon>
-      <SubType>Component</SubType>
-    </Compile>
-    <Compile Include="Synchronous\IStepQueryResponse.cs" />
-    <Compile Include="Asynchronous\QbManager.cs" />
-    <Compile Include="Synchronous\QbManager.cs" />
-    <Compile Include="Synchronous\QbWebConnectorSvc.cs">
-      <SubType>Component</SubType>
-    </Compile>
-  </ItemGroup>
   <ItemGroup>
-    <Content Include="Services\QbConnectorAsync.asmx" />
-    <Content Include="Services\QbConnector.asmx" />
-    <Compile Include="Services\QbConnector.asmx.cs">
-      <DependentUpon>QbConnector.asmx</DependentUpon>
-      <SubType>Component</SubType>
-    </Compile>
+    <PackageReference Include="Nito.AsyncEx" Version="5.0.0-pre-05" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
-    <None Include="WebConnector.nuspec" />
+    <ProjectReference Include="..\QbXml\QbXml.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\QbXml\QbXml.csproj">
-      <Project>{90be199d-5652-46b9-9c96-dfa9229b129c}</Project>
-      <Name>QbXml</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Messages\" />
+    <Reference Include="System.Web.Services">
+      <HintPath>..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7\System.Web.Services.dll</HintPath>
```

Comment:
incorrect path. Is there a NuGet package for this?

Reply:
There is not a Nuget for that API (see https://github.com/dotnet/core/issues/838).  I was manually linking in the 4.7 framework implementation.  Instead, I will target framework .Net 4.7.1 for WebConnector and it's tests.